WireAnalytics/Package.swift

```diff
@@ -20,12 +20,15 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/DataDog/dd-sdk-ios.git", exact: "2.12.0")
+        .package(url: "https://github.com/DataDog/dd-sdk-ios.git", exact: "2.12.0"),
+        .package(url: "https://github.com/Countly/countly-sdk-ios.git", exact: "24.4.2")
     ],
     targets: [
         .target(
             name: "WireAnalytics",
-            dependencies: resolveWireAnalyticsDependencies()
+            dependencies: resolveWireAnalyticsDependencies() + [
+                .product(name: "Countly", package: "countly-sdk-ios")
+            ]
```

Comment:
@agisilaos Adding Countly in the WireAnalytics is a good way to not ship it at all for C builds.

you'll need to introduce a COUNTLY_ENABLED env in CI.
+ cleanup the current flags we use to disable countly

Reply:
@netbe This is going to happen on a separate PR. For now I want to integrate Countly through SPM.